mozilla/bedrock
# Bug 1415544 - show QR code to alternative mobile app store for Fx Chi…

…na repack.

## Description
Certain pages show a QR code to Firefox at iOS App Store or Google Play Store, Beijing office would like to change it to provide the our customized build of Firefox for Android if an user visits such pages with Fx China repack.

We're using the url of Tencent app store since we believe WeChat is the QR code scanner of choice in China, and Tencent's own app store works best with it.

PR already updated after the merge of #5224.

## Issue / Bugzilla link
https://bugzil.la/1415544

## Testing
* Setup UITour permission for test 1
* Set "distribution.id" with Services.prefs.getDefaultBranch("distribution.").setCharPref("id", "MozillaOnline"); in Firefox's browser console.
* Visit /firefox/{54-56}/whatsnew/ or the future /firefox/mobile/, the QR code shown should link to http://a.app.qq.com/o/simple.jsp?pkgname=org.mozilla.firefox
